packages/components-react/link-react/src/link.test.tsx

```diff
@@ -1,13 +1,92 @@
 import { describe, expect, it } from '@jest/globals';
 import '@testing-library/jest-dom/jest-globals';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
+import { createRef } from 'react';
 import { Link } from './link';
 
+const href = '#top';
```

Comment:
```suggestion
const href = 'https://example.com/';
```

Anchor links / skip links zijn nog wat issues mee, laten we die niet als main test gebruiken